index.js

```diff
@@ -1 +1,38 @@
-console.log("Hello");
+const nodemailer = require("nodemailer");
+const express = require("express");
+require("dotenv").config();
+
+//express
+const app = express();
+
+const transporter = nodemailer.createTransport({
+  host: "smtp.gmail.com",
+  auth: {
+    user: process.env.USER,
+    pass: process.env.PASS,
+  },
+  secure: true,
+});
```

Comment:
why are using secure true here

Reply:
It is used because we are using the 2  factor auth and created a password explicitly for this application.